### PR TITLE
New changes for benchmark-comparison 

### PR DESCRIPTION
**Pull request by shahsahil264**
@paigerube14

### CODE REVIEW

**Comment by paigerube14** [comment]
Sahil, can you get a good passing build of this before we merge?

**Comment by paigerube14** [comment]
The output gets put into a csv file in the tmp folder, lets archive that file, but everything else looks good 

`file path name/tmp/cluster-density-v2-cebc6053-4e9c-405e-ab02-5f4f64164e6a/comparison.csv`

**Comment by qiliRedHat** [comment]
@shahsahil264 @paigerube14 I think BASELINE_UUID and GEN_CSV are still useful. For BASELINE_UUID, I may manually run this job to compare tow UUIDs I select instead of let the script to automatically get a UUID to compare to. For GEN_CSV, if I don't want to compare with any UUID but I still want to get a google sheets with the metrics, it is useful.

**Comment by paigerube14** [comment]
GEN_CSV is not currently a supported functionality of the new tooling I've added. Can be an enhancement. 
But I agree with Qiujie that BASELINE_UUID is useful for manual comparisons where we know the uuid to compare to

File: Jenkinsfile [hunk 1]
```diff
@@ -49,16 +39,6 @@ pipeline {
...
-            defaultValue: false, 
-            description: "Compare value with previous version"
-        )
-        string(
-          name: "COMPARE_WITH_PROFILE",
-          defaultValue: "",
-          description: 'Specify a profile you want to compare your job results to'
-        )
         string(
           name: "COMPARISON_CONFIG_PARAM",
```
Comment (paigerube14) [inline Jenkinsfile:43]: This and the tolerancy param are both old files/params that can be taken out

**Comment by shahsahil264** [comment]
> Sahil, can you get a good passing build of this before we merge?

This job passed: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/sahshah-e2e-benchmarking-multibranch-pipeline/job/benchmark-comparison/25/

**Pull request merged**